server/report_builder/README.md

```diff
@@ -312,6 +312,12 @@ Reports can be upserted using
 ./target/debug/remote_server_cli upsert-reports
 ```
 
+To overwrite any existing reports in the database, use the --overwrite flag
+
+```bash
+./target/debug/remote_server_cli upsert-reports --o
+```
+
```

Comment:
Is this relevant here? Added in this PR:
https://github.com/msupply-foundation/open-msupply/pull/6425

Reply:
Hmm doesn't really matter? Added it cause it wasn't there but now it's in a different PR so can just use the diff from 6425